vdr/cmd/cmd_test.go

```diff
@@ -280,7 +295,95 @@ func TestEngine_Command(t *testing.T) {
 			if !assert.Error(t, err) {
 				return
 			}
-			assert.Contains(t, buf.String(), "failed to deactivate DID document: server returned HTTP 404 (expected: 200)")
+			assert.Contains(t, errBuf.String(), "failed to deactivate DID document: server returned HTTP 404 (expected: 200)")
+		})
+	})
+
+	t.Run("addVerificationMethod", func(t *testing.T) {
+		id123, _ := did.ParseDID("did:nuts:123")
+		id123Method, _ := did.ParseDID("did:nuts:123#abc-method1")
+
+		pair, _ := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+		verificationMethod, _ := did.NewVerificationMethod(*id123Method, ssi.JsonWebKey2020, *id123, pair.PublicKey)
+
+		t.Run("ok", func(t *testing.T) {
+			cmd := newCmd(t)
```

Comment:
could you add the cool `t.Cleanup` to `newCmd` as well? With all the `Unsetenvs`?